packages/primitives/src/Icon.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { ComponentPropsWithRef } from "react";
+import { type RecipeVariantProps, cva, cx } from "@ndla/styled-system/css";
+
+export const iconRecipe = cva({
+  base: {
+    display: "inline-block",
+    fill: "currentcolor",
+    verticalAlign: "middle",
+    lineHeight: "1em",
```

Comment:
Bruke token her?

Reply:
Vi har ikke token for 1em. Tror det er ganske standard at ikoner skal ha 1em i line-height.